Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift

```diff
@@ -167,7 +167,7 @@ final class PledgePaymentMethodsViewController: UIViewController {
     let pledgePaymentMethodsValue = PledgePaymentMethodsValue(
       user: value.user,
       project: value.project,
-      applePayCapable: PKPaymentAuthorizationViewController.applePayCapable(for: value.project)
+      applePayDevice: PKPaymentAuthorizationViewController.applePayDevice()
```

Comment:
By usin `applePayDevice`, one can assume that we are returning a device with ApplePay, while we are returning a `Bool`. Maybe `deviceIsApplePayCapable` can be more self explanatory? 🤔